engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -34,12 +45,6 @@
 import org.jboss.logging.annotations.Message;
 import org.jboss.logging.annotations.MessageLogger;
 
-import static org.jboss.logging.Logger.Level.DEBUG;
-import static org.jboss.logging.Logger.Level.ERROR;
-import static org.jboss.logging.Logger.Level.INFO;
-import static org.jboss.logging.Logger.Level.TRACE;
-import static org.jboss.logging.Logger.Level.WARN;
-
```

Comment:
We agreed to have them at the beginning, so it's good to do it, and the wrong usages will go away over time. Only if you see a move in the opposite direction, then complain :)